geomesa-gt/geomesa-gt-partitioning/src/main/scala/org/locationtech/geomesa/gt/partition/postgis/dialect/package.scala

```diff
@@ -0,0 +1,426 @@
+/***********************************************************************
+ * Copyright (c) 2013-2022 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.gt.partition.postgis
+
+import com.typesafe.scalalogging.StrictLogging
+import org.locationtech.geomesa.gt.partition.postgis.dialect.PartitionedPostgisDialect.Config
+import org.locationtech.geomesa.utils.geotools.SimpleFeatureTypes
+import org.locationtech.geomesa.utils.geotools.SimpleFeatureTypes.AttributeOptions
+import org.locationtech.geomesa.utils.index.TemporalIndexCheck
+import org.locationtech.geomesa.utils.stats.IndexCoverage
+import org.opengis.feature.`type`.{AttributeDescriptor, GeometryDescriptor}
+import org.opengis.feature.simple.SimpleFeatureType
+
+import java.io.Closeable
+import java.sql.{Connection, PreparedStatement}
+import scala.util.Try
+import scala.util.control.NonFatal
+
+package object dialect {
+
+  private[dialect] val WriteAheadTableSuffix            = "_wa"
+  private[dialect] val PartitionedWriteAheadTableSuffix = "_wa_partition"
+  private[dialect] val PartitionedTableSuffix           = "_partition"
+  private[dialect] val AnalyzeTableSuffix               = "_analyze_queue"
+  private[dialect] val SortTableSuffix                  = "_sort_queue"
+
+  /**
+   * Wrapper to facilitate jdbc calls.
+   *
+   * Note: does not do any validation, inputs must be sanitized by the caller.
```

Comment:
Which inputs?

Reply:
all the inputs! specifically, any sql passed into the `execute` methods.

Reply:
updated to be clearer